packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -93,7 +106,13 @@ const metadata: Meta<TableArgs> = {
             WARNING - The table is still in development and considered
             experimental. It is not recommended for application use.
         </div>
-        <nimble-table :data=${x => dataSets[x.data]} id-field-name=${x => dataSetIdFieldNames[x.data]}>
+        <nimble-button @click="${x => x.populateData(x.tableRef, x.data)}">
+            Load data
+        </nimble-button>
+        <nimble-table
+            ${ref('tableRef')}
+            id-field-name="${x => dataSetIdFieldNames[x.data]}"
```

Comment:
Doing something like this worked for me locally to bind to the data set. Just abusing the binding a bit but then don't need the button

<img width="350" alt="image" src="https://user-images.githubusercontent.com/1588923/214737791-b6a42f2f-ee0b-42f7-9ce9-e45eed7ce913.png">